compiler/ilgen/OMRIlBuilder.hpp

```diff
@@ -346,13 +358,15 @@ class IlBuilder : public TR::IlInjector
    void Switch(const char *selectionVar,
                TR::IlBuilder **defaultBuilder,
                uint32_t numCases,
-               int32_t *caseValues,
-               TR::IlBuilder **caseBuilders,
-               bool *caseFallsThrough);
+               JBCase** cases);
    void Switch(const char *selectionVar,
                TR::IlBuilder **defaultBuilder,
                uint32_t numCases,
                ...);
+   JBCase * MakeCase(int32_t caseValue,
```

Comment:
In the spirit of improving documentation, would you please add a doxygen comment here and, if you're in a good mood, for two Switch services as well?

Reply:
What and excellent idea! :grin: